src/platform/windows/nvprefs/driver_settings.h

```diff
@@ -1,5 +1,13 @@
 #pragma once
 
+// nvapi headers
+// disable clang-format header reordering
```

Comment:
can you explain the reason why order matters in this case?

Reply:
<NvApiDriverSettings.h> need types from <nvapi.h>, but it does not include that file sadly. Clang-format swap the order of those files and it then does not compile.

Reply:
Sorry, I meant can you put it in a comment in the code? This will help contributors understand why it's like this in the future.

This would be fine for a comment: `// <NvApiDriverSettings.h> need types from <nvapi.h>`